Tests/Tests/STPPaymentContextAmountModelTest.m

```diff
@@ -0,0 +1,127 @@
+//
+//  STPPaymentContextAmountModelTest.m
+//  Stripe
+//
+//  Created by Ben Guo on 9/6/16.
+//  Copyright © 2016 Stripe, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "STPPaymentContextAmountModel.h"
+
+@interface STPPaymentContextAmountModelTest : XCTestCase
+
+@end
+
+@implementation STPPaymentContextAmountModelTest
```

Comment:
I think these tests should either be in `STPPaymentContextTest` or some of those should be in here, since there is some overlap.